pom.xml

```diff
@@ -1402,12 +1411,14 @@
                                 <version>2.8</version>
                                 <configuration>
                                     <links>
+                                        <link>http://docs.oracle.com/javase/8/docs/api/</link>
                                         <link>http://docs.oracle.com/javase/7/docs/api/</link>
                                         <link>http://java.sun.com/javase/6/docs/api/</link>
                                         <link>http://java.sun.com/javaee/5/docs/api/</link>
```

Comment:
Is there are reason to keep around links to the old JavaDoc versions, or should these old versions be retired out of these `<links>` with the introduction of newer versions?